ethcore/sync/src/chain/propagator.rs

```diff
@@ -48,7 +48,7 @@ fn accepts_service_transaction(client_id: &str) -> bool {
 	// Parity versions starting from this will accept service-transactions
 	const SERVICE_TRANSACTIONS_VERSION: (u32, u32) = (1u32, 6u32);
 	// Parity client string prefix
-	const PARITY_CLIENT_ID_PREFIX: &'static str = "Parity/v";
+	const PARITY_CLIENT_ID_PREFIX: &'static str = "Parity-Ethereum/v";
 
 	if !client_id.starts_with(PARITY_CLIENT_ID_PREFIX) {
```

Comment:
Are we seriously checking compatibility via version string? If yes, do you want me to check for both?

```
const LEGACY_CLIENT_ID_PREFIX: &'static str = "Parity/v";
const PARITY_CLIENT_ID_PREFIX: &'static str = "Parity-Ethereum/v";
if !client_id.starts_with(LEGACY_CLIENT_ID_PREFIX) && !client_id.starts_with(PARITY_CLIENT_ID_PREFIX) {
    // ...
}
```